core/aggregation.go

```diff
@@ -206,7 +206,6 @@ func (a *StdSignatureAggregator) ReceiveSignatures(
 				"operatorAddress", operatorAddr,
 				"socket", socket,
 				"batchHeaderHash", batchHeaderHashHex,
-				"attestationLatencyMs", r.AttestationLatencyMs,
```

Comment:
Any reason you are modifying this? This code doesn't even execute for v2 (same comment goes for the edit below)

Reply:
Unintentional, reverted